src/GSoft.Authentication.ClientCredentialsGrant.Tests/AuthenticationBuilderExtensionsTests.cs

```diff
@@ -1,10 +1,11 @@
-﻿using Microsoft.AspNetCore.Authentication;
+﻿using GSoft.AspNetCore.Authentication.ClientCredentialsGrant;
+using Microsoft.AspNetCore.Authentication;
 using Microsoft.AspNetCore.Authentication.JwtBearer;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Options;
 
-namespace GSoft.Extensions.Http.Authentication.ClientCredentialsGrant.Tests;
+namespace GSoft.Authentication.ClientCredentialsGrant.Tests;
```

Comment:
Some tests are still using the old namespace:

![image](https://user-images.githubusercontent.com/14242083/236907301-5e4f273a-31e9-4fc4-a4d7-9189e3ea22a4.png)

![image](https://user-images.githubusercontent.com/14242083/236907333-48dcde23-c040-4de8-bfda-c5c7a707863b.png)

Reply:
Oh you're right, I'll move them